src/objective-c/GRPCClient/GRPCCall.h

```diff
@@ -164,6 +164,12 @@ extern id const kGRPCTrailersKey;
 @interface GRPCCall : GRXWriter
 
 /**
+ * The server name for the RPC. If nil, the host name of the service object will be used instead.
```

Comment:
Please change this comment here per a5082db, something like: "The authority for the RPC. If nil, default authority will be used."